projects/frontend-ui/shared-components/.gitignore

```diff
@@ -0,0 +1,9 @@
+.idea/
+projects/gui/debug.log
+projects/gui/e2e/hars/
+projects/gui/e2e/logs/
+projects/gui/e2e/reports/
+projects/gui/e2e/screenshots/
+projects/gui/e2e/videos/
+projects/gui/reports/
+.DS_Store
```

Comment:
maybe reflect the directories we currently have, for example `projects/gui/e2e` would not have contents for now or should not exist at all, no reason to have this git-ignored